onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -1115,39 +1115,51 @@ Status MIGraphXExecutionProvider::Compile(const std::vector<FusedNodeAndGraph>&
     }
 
     std::vector<std::string> input_names, output_names;
-    no_input_shape = get_input_output_names(graph_body_viewer, input_names, output_names);
+    no_input_shape = no_input_shape or get_input_output_names(graph_body_viewer, input_names, output_names);
 
     // by parsing the model_proto, create a program corresponding to
     // the input fused_node
     migraphx::program prog;
 
     if (!no_input_shape) {
+      LOGS_DEFAULT(INFO) << "No Input shapes detected quantizing model" << std::endl;
```

Comment:
Personally I'd put this logging under VERBOSE as it's a step by step but that's just a nit.

Reply:
Figured its higher than verbose incase we get into some weird case where inputs aren't set for some reason